book/website/collaboration/ppie-events/ppie-events-checklist.md

```diff
@@ -0,0 +1,60 @@
+(cl-ppie-events-checklist)=
+# Checklist
```

Comment:
Is this something you will list out later - with the action points?

Reply:
Yes - it was hard to draft the checklist when the content kept changing so it is going to put on hold until the content has been finalized and signed off on :)